.github/workflows/build.yml

```diff
@@ -488,7 +488,7 @@ jobs:
       - name: Set up Deno
         uses: denoland/setup-deno@v1.1.4
         with:
-          deno-version: v1.38.5
+          deno-version: v1.x
```

Comment:
jsr wasn't supported on deno 1.38. deno.land/x/std/... won't receive non-critical updates and jsr is the recommended way

if needed we can lock it to some minor